ROMFS/px4fmu_common/init.d/4250_teal

```diff
@@ -115,60 +111,85 @@ then
 	param set MC_YAWRATE_D 0.0
 	param set MC_YAWRATE_MAX 180.0
 
-	param set MOT_SLEW_MAX 0.15
+	param set MOT_SLEW_MAX 0.15 # Set to reduce voltage transients as seen by battery management system.
 
-	# mc_pos_control
-	param set MPC_ACC_DOWN_MAX 10.0
-	param set MPC_ACC_HOR 10.0
-	param set MPC_ACC_HOR_MAX 15.0
-	param set MPC_ACC_UP_MAX 10.0
-	param set MPC_JERK_MAX 5.0
+	#### CONTROLLER ###
 	param set MPC_LAND_ALT1 8.0
 	param set MPC_LAND_ALT2 5.0
+	param set MPC_TKO_RAMP_T 0.75 # Position control smooth takeoff ramp time constant.
+	param set MPC_TKO_SPEED 0.75 # Takeoff climb rate.
+	param set MPC_TILTMAX_LND 18.0
+	param set MPC_TILTMAX_AIR 45.0
+	param set MPC_MAN_TILT_MAX 45.0
+
 	param set MPC_MANTHR_MAX 0.85
 	param set MPC_MANTHR_MIN 0.15
-	param set MPC_MAN_TILT_MAX 45.0
-	param set MPC_MAN_Y_MAX 200.0
-	param set MPC_THR_MAX 0.85
+	param set MPC_THR_MAX 0.85 # Full throttle can trip over current protection on BMS.
 	param set MPC_THR_MIN 0.15
-	param set MPC_TILTMAX_AIR 45.0
-	param set MPC_TKO_RAMP_T 0.75
-	param set MPC_TKO_SPEED 0.75
+
 	param set MPC_VEL_MANUAL 26.5
-	param set MPC_XY_CRUISE 15.0
+	param set MPC_XY_CRUISE 15.0 3 # RTL speed, it was too fast and scaring people.
+
+	param set MPC_MAN_Y_MAX 200.0 # Max manual yaw rate.
+
+	param set MPC_JERK_MAX 5.0
+	param set MPC_ACC_UP_MAX 10.0
+	param set MPC_ACC_DOWN_MAX 10.0
+	param set MPC_ACC_HOR 10.0
+	param set MPC_ACC_HOR_MAX 15.0
+
 	param set MPC_XY_P 1.15
 	param set MPC_XY_VEL_P 0.14
 	param set MPC_XY_VEL_I 0.014
 	param set MPC_XY_VEL_D 0.014
 	param set MPC_XY_VEL_MAX 26.5
-	param set MPC_Z_P 0.8
-	param set MPC_TILTMAX_LND 18.0
 
-	param set MPC_Z_VEL_D 0.02
-	param set MPC_Z_VEL_MAX_DN 2.5
-	param set MPC_Z_VEL_MAX_UP 6.0
+	param set MPC_Z_P 0.85 # default 1.0
+	param set MPC_Z_VEL_P 0.25 # default 0.2
+	param set MPC_Z_VEL_I 0.085 # default 0.02
+	param set MPC_Z_VEL_D 0.02 # default 0.0
+	param set MPC_Z_VEL_MAX_UP 20.0 # Documentation says limit is 8.0, but does not seem to be enforced in code.
+	param set MPC_Z_VEL_MAX_DN 2.5 # Probably could go lower than 2.5.
+	#### CONTROLLER ###
 
 	# navigator
-	param set NAV_ACC_RAD 2.5
-	param set NAV_RCL_ACT 1
+	param set NAV_ACC_RAD 2.5 # Acceptance radius for waypoint navigation.
+	param set NAV_RCL_ACT 1 # RC loss failsafe behavior -- hold mode.
 
 	# pwm control
 	param set PWM_DISARMED 900
 	param set PWM_MAX 1850
 	param set PWM_MIN 1075
-	param set PWM_RATE 400
+	param set PWM_RATE 0 # Oneshot125
 
 	# rtl
 	param set RTL_DESCEND_ALT 5
 	param set RTL_LAND_DELAY 5
 	param set RTL_MIN_DIST 7.5
 	param set RTL_RETURN_ALT 25
 
-	# sensors
-	param set SENS_EN_PGA460 1
-	param set SENS_EN_THERMAL 1
+	# calibration
+	param set CAL_ACC0_EN 1
+	param set CAL_ACC1_EN 0
+	param set CAL_ACC_PRIME 1442826 #mpu6500
 
-	# serial comms
-	param set SER_TEL2_BAUD 921600
+	param set CAL_GYRO0_EN 1
+	param set CAL_GYRO1_EN 0
```

Comment:
Why disable accel and gyro 1?

Reply:
That is the mpu9250 gyro/accel which is hard mounted to the board.

Reply:
Oh weird, I didn't realize you wanted to completely ignore its accel and gyro. I thought it would be a backup.

Reply:
When we tested in it ended up blending them and the quad flew like :poop:

Reply:
It definitely doesn't blend them unless something is very wrong (failing back and forth). This is fine for now, but I'd strongly recommend revisiting it as a backup IMU later.